src/modules.js

```diff
@@ -61,14 +61,17 @@ var LibraryManager = {
       'library_formatString.js',
       'library_math.js',
       'library_path.js',
-      'library_syscall.js',
       'library_html5.js',
       'library_stack_trace.js',
-      'library_wasi.js',
       'library_int53.js',
       'library_dylink.js'
     ];
 
+    if (!WASMFS) {
+      libraries.push('library_syscall.js')
+      libraries.push('library_wasi.js')
+    }
```

Comment:
Given that we realized this files contain non-filesystem syscalls too, we might want to remove this if. Things should still work for you, although the error messages might be less clear - basically, any time you see `var FS` being included, that'd mean you have a syscall you need to implement. But, if the error messages are useful, then given this is still an experimental option that seems fine to me.

Reply:
I suggest instead you either:

1) modify these files and wrap the FS syscalls in `if !WASMFS`
2) split these out into `library_wasi_fs.js` and `'library_syscall_fs.js'